.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Can I find out why? ;)
I have big dilemma about argumentation about 2 spaces as indent in my company.